packages/libs/ui/SceneryTownSource.tsx

```diff
@@ -65,9 +65,13 @@ export const SceneryTownSource = (props: SceneryTownSourceProps) => {
   const filter: ExpressionSpecification =
     game === 'ats'
       ? [
-          'in',
-          ['get', 'state'],
-          ['literal', [...(props.enabledStates ?? allStates)]],
+          'all',
+          ['boolean', ['get', 'show'], true],
```

Comment:
Could you repeat the comment in `CitySearchBar.tsx`?

```suggestion
          // specify `true` as a fallback so we don't skip labels with undefined `show`
          ['boolean', ['get', 'show'], true],
```